cmake/modules/FindUSD.cmake

```diff
@@ -158,8 +158,7 @@ include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(USD
     REQUIRED_VARS
         PXR_USD_LOCATION
-        USD_INCLUDE_DIR
-        USD_LIBRARY_DIR
+        PXR_INCLUDE_DIRS
```

Comment:
Use this var which comes from pxrConfig.cmake (which we include above) rather than our own USD_INCLUDE_DIR which came from find a header.